### PR TITLE
uadk: update the symbol table for libraries

### PR DESCRIPTION
**Pull request by Liulongfang**
1. add dlopen error log.
2. update the libraries function symbol.

### CODE REVIEW

**Comment by Liulongfang** [comment]
> 

OK， thanks.
2. I check and deal with it.
3. Yes, if you need to add an external API, you need to add it in the corresponding x.map

**Comment by gaozhangfei** [comment]
+1

**Comment by hzhuang1** [comment]
Which kind of issue do you want to fix?

**Comment by Liulongfang** [comment]
This is a function required by internal security requirements iteration, protecting the external interface symbol table of the so library file

File: libwd.map [hunk 1]
```diff
@@ -0,0 +1,39 @@
+UADK_2.0 {
```
Comment (hzhuang1) [inline libwd.map:1]: I prefer to follow the naming policy as "LIBWD_2.0".

https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html

Comment (Liulongfang) [reply libwd.map:1]: We mainly reflect the UADK framework now, so the corresponding lib library interface version name is declared as UADK_2.0

**Pull request merged**